### DISCUSSION TITLE
returned non-zero exit status 1

### DISCUSSION THREAD

**Discussion by deleep225** [deepmodeling/dpgen#605] [General]
Im trying to do dpgen example for al in **init** process.

I am able to do till first principle simulations. all the FPMD are successful, but dpgen is unable to collect the data.
following is the error.

Traceback (most recent call last):
  File "/rhome/dhileept/.local/bin/dpgen", line 8, in <module>
    sys.exit(main())
  File "/rhome/dhileept/.local/lib/python3.9/site-packages/dpgen/main.py", line 175, in main
    args.func(args)
  File "/rhome/dhileept/.local/lib/python3.9/site-packages/dpgen/data/gen.py", line 795, in gen_init_bulk
    coll_vasp_md(jdata)
  File "/rhome/dhileept/.local/lib/python3.9/site-packages/dpgen/data/gen.py", line 541, in coll_vasp_md
    raise RuntimeError("MD dir: %s: find no valid outcar in sys %s, "
RuntimeError: MD dir: /rhome/dhileept/tut_dpgen/dpgen/examples/init/al/al.POSCAR.01x01x01/02.md: find no valid outcar in sys sys-0004, check if your vasp md simulation is correctly done

If anyone faced the above problem, please help.
Thank you.

**Comment by deleep225** [comment]
dpgen.log shows the below message
 WARNING : in directory /rhome/dhileept/tut_dpgen/dpgen/examples/init/al/al.POSCAR.01x01x01/02.md/sys-0004 nforce in OUTCAR is not equ

**Reply by sunlight905** [reply]
Do you have solved this problem? I have meet the same problem.

**Comment by biglinn** [comment]
I have met this probelm too. Do you know what is the reason for that?
I have kept 'md_nsteps' in param.json and 'NSW' in INCAR in agreement as the tourial tells me. However, there is still this problem.
I dont know what "find no valid OUTCAR" means.
